src/content/docs/journey-migration/migrating-infra.mdx

```diff
@@ -0,0 +1,230 @@
+
+import journeyNewDash from 'images/infrastructure_screenshot-crop_journey-network-dash.png'
+
+import journeyMainDash from 'images/infrastructure_screenshot-crop_migration-dashboard.png'
+
+Monitoring your infrastructure can be a seemingly daunting task at first glance. Gathering the data from your servers, databases, cloud platforms, and hosted services is only the first task, and once you've set all that up, you still need to find ways to make sense of it all. 
+
+That's where New Relic comes in. Not only do our guided installs and pre-configured dashboards and alerts help you get up and running quickly, our tools will help you proactively troubleshoot and improve your infrastructure. Whether you want to view one or one-hundred of your hosts, you can do everything you need from one central location without having to 
+
+<img
+    title="infrastructure_dashboards"
+    alt="An image displaying New Relic's infrastructure monitoring dashboard"
+    src={journeyMainDash}
+/>
+
+## Your infrastructure monitoring solution [#infra-solution]
+
+New Relic makes it easy for you to observe your entire infrastructure, whether it runs in the cloud or dedicated hosts. In Datadog, you may be familiar with these capabilities as `agents`, and we call the same infrastructure capabilities `agents` at New Relic as well. Here are a few of the offerings for New Relic infrastructure monitoring:
+
+* Infrastructure agents for Linux, macOS, and Windows operating systems with guided and manual install options.
+* An agent for for [Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/) that can be set up in a number of ways.
+* Cloud agents for Amazon, Azure, and Google Cloud Platform that you can install independently of our infrastructure agents.
+* On-host integration agents for services like MySQL, Apache, Docker, and Kafka that work together with our infrastructure agent to maximize your data reporting.
+
+As long as your system and on-host integrations meet our [minimum requirements](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/), all you need to try is a [free New Relic account](https://newrelic.com/signup) and you can start right away.
+
+## Get started with New Relic infrasctructure monitoring [#get-started]
+
+Your infrastructure data in New Relic is located in the [Infrastructure](https://one.newrelic.com/infra) section of the UI, exactly the same as Datadog. The fastest way to get an infrastructure monitor up and running can be done through our [guided installation](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==) path, which you can find within [New Relic](https://one.newrelic.com) by selecting **Add Data > All > Guided install**.
```

Comment:
This first UI link should contain `one.newrelic.com` in the link text per pathing guidelines https://docs.newrelic.com/docs/style-guide/formatting/ui-paths/

Reply:
Ditto the `within New Relic` one

Reply:
Fixed!